backend/kale/common/snapshotutils.py

```diff
@@ -0,0 +1,119 @@
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  To allow a notebook to create Rook CephFS snapshot a ClusterRole
+#  and RoleBinding for the default-editor service account in the
+#  given namespace must be applied. Below are examples for use with the
+#  namespace "admin".
+#
+#  !!!WARNING!!!
+#  Might not be secure, only use for testing
+#
+#  apiVersion: rbac.authorization.k8s.io/v1
+#  kind: ClusterRole
+#  metadata:
+#    name: snapshot-access
+#  rules:
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshots"]
+#      verbs: ["create", "get", "list", "watch", "patch", "delete"]
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshotcontents"]
+#      verbs: ["create", "get", "list", "watch", "update", "delete"]
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshotclasses"]
+#      verbs: ["get", "list", "watch"]
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshotcontents/status"]
+#      verbs: ["update"]
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshots/status"]
+#      verbs: ["update"]
+#    - apiGroups: ["snapshot.storage.k8s.io"]
+#      resources: ["volumesnapshotclasses"]
+#      verbs: ["get", "list", "watch"]
+#    - apiGroups: ["storage.k8s.io"]
+#      resources: ["storageclasses"]
+#      verbs: ["get", "list"]
+#
+#  apiVersion: rbac.authorization.k8s.io/v1
+#  kind: RoleBinding
+#  metadata:
+#    name: allow-snapshot-nb-admin
+#    namespace: admin
+#  subjects:
+#  - kind: ServiceAccount
+#    name: default-editor
+#    namespace: admin
+#  roleRef:
+#    kind: ClusterRole
+#    name: snapshot-access
+#    apiGroup: rbac.authorization.k8s.io
```

Comment:
In general, if you want to provide documentation for a module, use triple quotes. Look up "modules docstrings" to see some examples. Also

Reply:
Also, this doesn't seem the right place to document these things. If you feel like these are important let's have something under an `examples` folder or similar

Reply:
Agreed, it was more a development comment for others. Should probably create a README for this with some other info instead.